app/embeds/GoogleDrawings.test.js

```diff
@@ -0,0 +1,29 @@
+/* eslint-disable flowtype/require-valid-file-annotation */
+import GoogleDrawings from "./GoogleDrawings";
+
+describe("GoogleDrawings", () => {
+  const match = GoogleDrawings.ENABLED[0];
+  test("to be enabled on share link", () => {
```

Comment:
What about links like: ```https://docs.google.com/drawings/d/XXX/view?usp=sharing```.

Reply:
I am not sure. Google provides a link to copy when publishing the file and the link ends with `/edit...`.

Reply:
From my latest researches in this topic, I have got that Google does add always the "usp" parameter when creating a shared URL. The parameter can have multiple values.

So, checking in the regex on the usp parameter would be good.

Reply:
> i explicit want to allow links without the parameter to be able to integrate drawings that are not published.

as I mentioned above